drupal-ukraine/advanced_forum
# Vm d8

- added cibox vm
- drupal 7
- drupal 8 (no automation there, just stable codebase)

Steps for review
- download branch vm_d8
- vagrant up 
- check vm is working
- check D7 (docroot, reinstall.sh) and D8 ( http://drupal.192.168.56.132.xip.io/d8/core/install.php ) can be installed
- merge
